src/components/drag_and_drop/draggable.tsx

```diff
@@ -104,6 +110,14 @@ export const EuiDraggable: FunctionComponent<EuiDraggableProps> = ({
               data-test-subj={dataTestSubj}
               className={classes}
               style={{ ...style, ...provided.draggableProps.style }}
+              role={
+                draggableContainer ? 'group' : provided.dragHandleProps?.role
+              }
+              tabIndex={
```

Comment:
Minor code comment requests - your GH comments were so useful in providing context I think they should be immortalized in code :)

```suggestion
              // We use [role="group"] instead of [role="button"] when we expect a nested
              // interactive element. Screen readers will cue users that this is a container and
              // has one or more elements inside that are part of a related group.
              role={
                draggableContainer ? 'group' : provided.dragHandleProps?.role
              }
              // If the container includes an interactive element, we remove the [tabindex=0]
              // because [role="group"] does not permit or warrant a tab stop
              tabIndex={
```